src/Elliptic/Systems/Xcts/BoundaryConditions/Flatness.hpp

```diff
@@ -0,0 +1,145 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace Xcts::BoundaryConditions {
+namespace detail {
+
+struct FlatnessImpl {
+  using options = tmpl::list<>;
+  static constexpr Options::String help =
+      "Impose flat spacetime at this boundary.";
+
+  using argument_tags = tmpl::list<>;
+  using volume_tags = tmpl::list<>;
+
+  static void apply(gsl::not_null<Scalar<DataVector>*> conformal_factor,
```

Comment:
Sorry, I haven't been following the other features of the Xcts system -- can you explain this interface? I would have anticipated the arguments to have something to do with the tag lists, but perhaps these are the 'standard' system variables for Xcts?

Reply:
You should find all answers in the docs of the base class (I hope): https://spectre-code.org/classelliptic_1_1BoundaryConditions_1_1BoundaryCondition.html :) Let me know if the docs leave anything unclear.